.github/workflows/release.yml

```diff
@@ -21,6 +21,18 @@ jobs:
       - name: Set tag variable
         run: echo "TAG=$(echo ${GITHUB_REF:10})" >> $GITHUB_ENV
 
+      - name: Retrieve date from git revision
+        id: git_date
+        run: |
+          DATE=$(git log -1 --format=%cd --date=format:%Y-%m-%d || echo "NoCommits")
+          if [ "$DATE" = "NoCommits" ]; then
+            RELEASE_DATE=""
+          else
+            RELEASE_DATE="$DATE"
+          fi
+          echo "KIWIX_ANDROID_RELEASE_DATE=$RELEASE_DATE" >> $GITHUB_ENV
```

Comment:
please don't echo here in workflows.

Reply:
@kelson42 It is for setting the env variable, not for printing its value [see](https://github.com/kiwix/kiwix-android/actions/runs/12608455860/job/35140926560#step:5:8), without `echo` it will not store the output of "RELASE_DATE". However, this is a non-critical env variable so github printing it in the next job [see](https://github.com/kiwix/kiwix-android/actions/runs/12608455860/job/35140926560#step:6:7).